docs/lsm/attributes_metadata/attributes_metadata.rst

```diff
@@ -85,3 +85,90 @@ Supported values
 
 
 Attributes modifiers can also be specified on :ref:`relational attributes<attribute_modifiers_on_a_relationship>`.
+
+
+------------
+
+Annotations
+~~~~~~~~~~~
+
+Definition
+##########
+
+Annotations are key-value pairs that can be associated with an entity (service entity or embedded entity) or
+an attribute (simple attribute or relational attribute). These annotations are intended for the web-console to
+improve visualizations and the user-experience.
```

Comment:
```suggestion
Annotations are key-value pairs that can be associated with an entity (service entity or embedded entity) or an attribute (simple attribute or relational attribute). These annotations  don't influence the behavior of LSM or the Inmanta Service Orchestrator itself, but are intended to pass meta data to other components. For example, they can be used to pass on visualization meta-data to the the web-console to improve the user-experience.
```

not sure about this, just a suggestion.

Reply:
I wasn't sure how to write this part, because the use cases are open ended at the moment. I like the suggestion,